src/views/HomeView/index.js

```diff
@@ -3,7 +3,7 @@ import React, { PropTypes } from 'react'
 import { connect } from 'react-redux'
 import { increment, doubleAsync } from '../../redux/modules/counter'
 import DuckImage from './Duck.jpg'
-import classes from './HomeView.scss'
+import classes from './style.scss'
```

Comment:
I didn't have that issue because I use BEM notation anyway, so inside HomeView/style.css I mainly have .HomeView {} classes